src/app/dotnet-fake/dotnet-fake.fsproj

```diff
@@ -1,30 +1,31 @@
-<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
-    <DefineConstants>$(DefineConstants);CORE_CLR;DOTNETCORE;EXPLICIT_DEPENDENCIES;NETSTANDARD;NETSTANDARD1_6</DefineConstants>
-    <DebugType>Portable</DebugType>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <AssemblyName>dotnet-fake</AssemblyName>
-    <OutputType>Exe</OutputType>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
-    <DefineConstants>$(DefineConstants);RELEASE</DefineConstants>
-  </PropertyGroup>
-  <ItemGroup>
-    <Compile Include="AssemblyInfo.fs" />
-    <Compile Include="..\Fake.netcore\Cli.fs" />
-    <Compile Include="..\Fake.netcore\Program.fs" />
-    <!-- Note: This resource should be moved to Paket.Core... -->
-    <EmbeddedResource Include="..\..\..\.paket\Paket.Restore.targets" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Fake.Core.CommandLineParsing\Fake.Core.CommandLineParsing.fsproj" />
-    <ProjectReference Include="..\Fake.Runtime\Fake.Runtime.fsproj">
-      <FromP2P>true</FromP2P>
-    </ProjectReference>
-    <ProjectReference Include="..\Fake.Core.Context\Fake.Core.Context.fsproj">
-      <FromP2P>true</FromP2P>
-    </ProjectReference>
-  </ItemGroup>
-  <Import Project="..\..\..\.paket\Paket.Restore.targets" />
-</Project>
\ No newline at end of file
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" 
+    xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
I think we should consider leaving netcoreapp2.0 (or both) here

Reply:
I will change that later today when i am at home. Do you want both or just 2,0?

Reply:
Thanks! I guess leaving 2.0 for now is the better option. We just wait until people start complaining.